Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -17,6 +17,31 @@ public final class ProjectPamphletViewController: UIViewController {
 
   @IBOutlet weak private var navBarTopConstraint: NSLayoutConstraint!
 
+  private let backThisProjectContainerViewMargins = Styles.grid(3)
+  private let backThisProjectContainerView: UIView = {
+    return UIView() |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  }()
+
+  private let backThisProjectButton: UIButton = {
+     return UIButton(type: .custom) |> \.translatesAutoresizingMaskIntoConstraints .~ false
```

Comment:
Wonder if by using `.custom` the button does need explicitly setting selected/highlighted state color. Right now tapping and holding on the button doesn't give user a hint that anything is happening.

Reply:
Yeah with `.custom` you have to set the highlighted color explicitly. I just set it to the same color because I'm not sure what color we want :\ the system behavior seemed too dramatic - I was planning on getting some design feedback since I want this to be the same styling we use for the buttons on the reward cells.

Reply:
@dusi I'm updating the `.highlighted` and `.selected` background color to just be the same color with an alpha of 0.8 for now - we can change it later during a design sweep if we want to

Reply:
For what it's worth, on https://github.com/kickstarter/ios-private/pull/438, I used a `.custom` close button and seem to get the highligh state for free 🤔

Reply:
Yeah, I think we figured out in one of the comments bellow that it's due to the fact that both `.normal` & `.highlighted` colors are the same

Reply:
Updated to use `greenButtonStyle` from `ButtonStyles.swift` which has highlighted state handling!